geode-connectors/src/distributedTest/java/org/apache/geode/connectors/jdbc/internal/cli/CreateMappingCommandDUnitTest.java

```diff
@@ -176,15 +181,14 @@ private static void validateRegionAlteredOnServer(String regionName, boolean syn
   }
 
   @Test
-  @Parameters({REGION_NAME, "/" + REGION_NAME})
+  @Parameters({TEST_REGION, "/" + TEST_REGION})
   public void createMappingUpdatesServiceAndClusterConfig(String regionName) {
     setupReplicate(regionName);
     CommandStringBuilder csb = new CommandStringBuilder(CREATE_MAPPING);
-    csb.addOption(CREATE_MAPPING__REGION_NAME, regionName);
-    csb.addOption(CREATE_MAPPING__DATA_SOURCE_NAME, "connection");
-    csb.addOption(CREATE_MAPPING__TABLE_NAME, "myTable");
-    csb.addOption(CREATE_MAPPING__PDX_NAME, "myPdxClass");
-    csb.addOption(CREATE_MAPPING__ID_NAME, "myId");
```

Comment:
It looks like this line about "myId" was dropped. I think we want to keep it. Also lines 196 and 206 were dropped. This may have happened when resolving conflicts?